src/containers/treePage/index.tsx

```diff
@@ -33,14 +33,21 @@ import { RecordStewardshipRequest } from '../../components/forms/ducks/types';
 import useWindowDimensions, {
   WindowTypes,
 } from '../../components/windowDimensions';
-import ReturnButton from '../../components/returnButton';
 import TreeInfo from '../../components/treeInfo';
 import TreeActivity from '../../components/treeActivity';
 import EntryList from '../../components/entryList';
 import { CenterDiv } from '../../components/themedComponents';
+import { STREET_ZOOM } from '../../components/mapPageComponents/constants';
 
 const { Title } = Typography;
 
+const ReturnButton = styled(Button)`
```

Comment:
Can we give the link button an optional state argument? that way we don't need to defined this style or link functionality below more than once. Plus I'm sure we'll wind up reusing this for the blocks in the future.